lib/linguist/languages.yml

```diff
@@ -766,6 +766,32 @@ Charity:
   tm_scope: none
   ace_mode: text
   language_id: 56
+Checksums:
+  type: data
+  aliases:
+    - hash
+    - hashes
+    - sum
+    - sums
+  filenames: [
+    "sum",      "sums",
```

Comment:
Names must be sorted alphabetically. Also, all entries use the bullet syntax for lists, instead of `[...]`.

Reply:
I originally wanted to group names by hash type, but I've changed it to meet that requirement.

Reply:
@theonlypwner Trouble is, some of these filenames might not meet the usage requirements, which means they'll end up being removed (forcing you to regroup or reformat the remaining entries).